src/shared/utils/loginProviders.ts

```diff
@@ -4,8 +4,49 @@ import isString from 'lodash/isString'
 import bitbucketLogo from 'assets/providers/bitbucket-icon.svg'
 import githubLogo from 'assets/providers/github-icon.svg'
 import gitlabLogo from 'assets/providers/gitlab-icon.svg'
+import oktaLogo from 'assets/providers/okta-icon.svg'
 import sentryLogo from 'assets/providers/sentry-icon.svg'
 
+export const LoginProvidersEnum = {
+  BITBUCKET: {
+    provider: 'bb',
+    name: 'Bitbucket',
+    external: 'BITBUCKET',
+    externalKey: 'bb',
+    selfHosted: 'BITBUCKET_SERVER',
+    selfHostedKey: 'bbs',
+    selfHostedName: 'Bitbucket Server',
+    variant: 'bitbucket',
+  },
+  GITHUB: {
+    provider: 'gh',
+    name: 'GitHub',
+    external: 'GITHUB',
+    externalKey: 'gh',
+    selfHosted: 'GITHUB_ENTERPRISE',
+    selfHostedKey: 'ghe',
+    selfHostedName: 'GitHub Enterprise',
+    variant: 'github',
+  },
+  GITLAB: {
+    provider: 'gl',
+    name: 'GitLab',
+    external: 'GITLAB',
+    externalKey: 'gl',
+    selfHosted: 'GITLAB_ENTERPRISE',
+    selfHostedKey: 'gle',
+    selfHostedName: 'GitLab CE/EE',
+    variant: 'gitlab',
+  },
+  OKTA: {
+    provider: 'okta',
+    name: 'Okta',
+    external: 'OKTA',
+    externalKey: 'okta',
```

Comment:
Is this object meant to have self-hosted information? Would think it's odd for self-hosted to want to log in w/ okta, but just seeing inconsistency bw/ these objects 😬 I guess that's fine